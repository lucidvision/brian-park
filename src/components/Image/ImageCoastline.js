import React from 'react'
import coastlineMobile from 'assets/coastline-mobile.png'

export default function ImageCoastline (props) {
  return (
    <img src={coastlineMobile} alt='Coastline Market Mobile App' {...props} />
  )
}
