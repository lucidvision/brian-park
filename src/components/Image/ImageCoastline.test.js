import React from 'react'
import { shallow } from 'enzyme'
import ImageCoastline from './ImageCoastline'
import coastlineMobile from 'assets/coastline-mobile.png'

describe('ImageCoastline', () => {
  const imageCoastline = shallow(<ImageCoastline />)

  it('renders correctly', () => {
    expect(imageCoastline).toMatchSnapshot()
  })

  it('renders img with the correct props', () => {
    expect(imageCoastline.find('img').props()).toEqual({
      src: coastlineMobile,
      alt: 'Coastline Market Mobile App'
    })
  })
})
