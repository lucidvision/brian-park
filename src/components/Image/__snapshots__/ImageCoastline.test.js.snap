// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCoastline renders correctly 1`] = `
<img
  alt="Coastline Market Mobile App"
  src="coastline-mobile.png"
/>
`;
